.travis.yml

```diff
@@ -197,6 +200,8 @@ script:
         PACKAGE_DEPLOY=$(greadlink -f OpenMEEG*.tar.gz)
       fi
     fi
+  - sudo make install
+  - sudo make uninstall
```

Comment:
Why those two ?

Reply:
to test uninstall you need to install first.

Reply:
I had to review, why we did that. (lets see how many times do I get to this comment)